### PR TITLE
Maas 2979 info about data series in alert

### PR DESCRIPTION
**Pull request by tetianakh**
Extracted trigger result evaluation logic into a separate class to simplify testing.

Graphite plugin result now contains data from each individual metric.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,5 +9,5 @@ cchardet==1.1.3
 pydruid==0.3.1
 python-consul==0.6.2
 stevedore==1.23.0
-git+https://github.com/RulersOfAsgard/alamo-common.git@master#egg=alamo_common
+git+https://github.com/RulersOfAsgard/alamo-common.git@7e713079cd8d2ffba3906#egg=alamo_common
```
Comment (h00dy) [inline requirements.txt]: Do we really want to fetch specific commit version of alamo-comon?

Comment (tetianakh) [reply requirements.txt]: No, we don't. It was the only way to run tests before the pull request #21 in alamo-common was merged. I'm reverting this change now.

File: requirements.txt [hunk 2]
```diff
@@ -9,5 +9,5 @@ cchardet==1.1.3
 pydruid==0.3.1
 python-consul==0.6.2
 stevedore==1.23.0
-git+https://github.com/RulersOfAsgard/alamo-common.git@7e713079cd8d2ffba3906#egg=alamo_common
+git+https://github.com/RulersOfAsgard/alamo-common.git@develop#egg=alamo_common
```
Comment (h00dy) [inline requirements.txt]: Why not master?

Comment (tetianakh) [reply requirements.txt]: because it's not on master yet https://github.com/RulersOfAsgard/alamo-common/pull/22

**Pull request merged**